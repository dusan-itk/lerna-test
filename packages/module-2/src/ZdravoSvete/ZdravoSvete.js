import React from 'react';

import './ZdravoSvete.scss';

export function ZdravoSvete() {
  return <h1 className="zdravo-svete">Zdravo svete od modula 2</h1>;
}
