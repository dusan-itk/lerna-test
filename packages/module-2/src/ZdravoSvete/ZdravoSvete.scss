$test: blue;

.zdravo-svete {
  background-color: $test;
  padding: 10px 20px;
}
