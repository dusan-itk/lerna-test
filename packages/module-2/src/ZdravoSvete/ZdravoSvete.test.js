import React from 'react';
import { mount } from 'enzyme';

import { ZdravoSvete } from './ZdravoSvete';

describe('ZdravoSvete', () => {
  it('shoud match snapshot', () => {
    expect(mount(<ZdravoSvete />)).toMatchSnapshot();
  });
});
