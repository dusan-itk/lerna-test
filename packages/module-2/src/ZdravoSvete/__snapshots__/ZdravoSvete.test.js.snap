// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ZdravoSvete shoud match snapshot 1`] = `
<ZdravoSvete>
  <h1
    className="zdravo-svete"
  >
    Zdravo svete od modula 2
  </h1>
</ZdravoSvete>
`;
